allennlp/models/semantic_role_labeler.py

```diff
@@ -51,6 +55,8 @@ def __init__(self, vocab: Vocabulary,
         self.stacked_encoder = stacked_encoder
         self.tag_projection_layer = TimeDistributed(Linear(self.stacked_encoder.get_output_dim(),
                                                            self.num_classes))
+        initializer(self.stacked_encoder)
```

Comment:
We need to be careful doing that, because it will also initialize our carefully pre-initialized embedding layers. Given this model is simple, I didn't use the regex functionality of the `InitializerApplicator`, but I can do if you would prefer.